File: platforms/web/examples/core/renderer/create_3d_texture.js
const r = new Renderer();
const size = fragmentcolor.Size { width: 16, height: 16, depth: Some(8) };
const voxels = [0u8; (16*16*8*4) as usize];
const tex = futures.executor.blockOn(;
    r.create3dTexture(voxels, Some(size), Some(wgpu.TextureFormat.Rgba8Unorm));
).unwrap();

File: platforms/python/examples/core/renderer/create_3d_texture.py
r = Renderer()
size = fragmentcolor.Size { width: 16, height: 16, depth: Some(8) }
voxels = [0u8; (16*16*8*4) as usize]
tex = futures.executor.block_on(
    r.create_3d_texture(voxels, Some(size), Some(wgpu.TextureFormat.Rgba8Unorm))
).unwrap()